### PR TITLE
adding set subscription group method

### PR DESCRIPTION
**Pull request by nickrobin2**
extending the functionality here to allow customers to set a subscription group ID

### CODE REVIEW

**Comment by davidbielik** [comment]
would be good to add the SDK minimum versions for each, but otherwise looks good!

File: braze-templates/4-sms-capture-modal/index.html [hunk 1]
```diff
@@ -571,6 +572,7 @@ <h1 class="modal__title">Thanks for signing up!</h1>
         });
     </script>
     <script>
+        var subscriptionGroupId = "ENTER YOUR SUBSCRIPTION GROUP ID";
```
Comment (davidbielik) [inline braze-templates/4-sms-capture-modal/index.html]:
```suggestion
        var subscriptionGroupId = "ENTER YOUR SUBSCRIPTION GROUP ID FOUND IN DASHBOARD";
```

File: braze-templates/4-sms-capture-modal/README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+> ⚠️ Note: Subscription Groups were introduced in Android SDK v15.0.0, Web SDK v3.4.0, and iOS SDK v4.3.3. Please ensure your SDKs are up to date, or apply filters to your message's targeting.
```
Comment (davidbielik) [inline braze-templates/4-sms-capture-modal/README.md:1]: @nickrobin how's this?

Comment (nickrobin2) [reply braze-templates/4-sms-capture-modal/README.md:1]: OK - that works for me ! Thanks for pulling the exact versions!

**Pull request merged**